server/status_test.go

```diff
@@ -46,18 +46,37 @@ func startStatusServer() *httptest.Server {
 	return httpServer
 }
 
-// TestStatusLocalStacks verifies that goroutine stack traces are available
-// via the /_status/local/stacks endpoint.
-func TestStatusLocalStacks(t *testing.T) {
+// TestStatus verifies that status endpoints return expected results.
+func TestStatus(t *testing.T) {
 	s := startStatusServer()
 	defer s.Close()
-	body, err := getText(s.URL + statusLocalStacksKey)
-	if err != nil {
-		t.Fatal(err)
+
+	testCases := []struct {
+		keyPrefix string
+		expected  string
+	}{
+		{statusKeyPrefix, "{}"},
```

Comment:
The test is a little weird because it's not actually using the other content-types. It's verifying that we always return json no matter what the accept header says (this is worth testing because an earlier version of this PR would panic if the client sent `Accept: application/x-protobuf`). To that end, it should also verify that the response content-type is application/json.